### ISSUE TITLE
about dataset

### ISSUE THREAD

**Issue by 1607722534** [yusanshi/NewsRecommendation#22]
我看到有论文中把MIND-small val数据集中10%当验证集，90%当测试集，这样做比直接复制验证集当测试集合理吗？

**Comment by yusanshi** [comment]
是的我觉得这样做或者把 training set 拆成 training set 和 validation set，都比 README 里面所做的直接复制 validation set 为 test set 更好一些
